src/main/java/org/kuali/kfs/sys/document/web/renderers/GroupTitleLineRenderer.java

```diff
@@ -362,10 +358,10 @@ protected void renderHideDetails(PageContext pageContext, Tag parentTag) throws
     }
 
     /**
-     * Oy, the big one...this one actually renders instead of returning the HTML in a String. This is because it's kind of complex
-     * (and a likely target for future refactoring)
+     * Oy, the big one...this one actually renders instead of returning the HTML in a String. This is because it's kind 
+     * of complex(and a likely target for future refactoring)

```

Comment:
This line doesn't appear to have all of the comment-formatting changes from the new KualiCo release.

Reply:
I double checked, and this is how the comment is setup, when I look at the KualiCo version  of GroupTitleLineRenderer.java from Feb 15

Reply:
According to the GitHub history for the Feb 15 patch, this line has a space before the parentheses and a period at the end. Not sure what's causing the discrepancy you're seeing.

Given that this line is just a comment and not actual code, I'm okay with leaving it as-is if needed.

Reply:
interesting, I am using eclipse to open the class file from 2/15 jar